client/src/components/Posts/index.js

```diff
@@ -9,19 +9,25 @@ import NextPageButton from '../NextPageButton';
 
 const Posts = () => {
   const posts = useSelector(state => state.posts);
-
-  return (
-    <div className='Posts' id='Posts'>
-
-        {(page > 0) ? <PrevPageButton /> : ''}
-
-        {posts.map(post => (
-          <Post data={post} key={post.url} />
-        ))}
-
-        {(!lastPage) ? <NextPageButton /> : ''}
-    </div>
-  );
+  if (posts.length > 0) {
+    return (
+      <div className='Posts' id='Posts'>
```

Comment:
Is the id attribute used somewhere else?

Reply:
No, I just kept it there for posterity